docs/src/content/pages/fields/slug.mdoc

```diff
@@ -23,6 +25,17 @@ title: fields.slug({
 })
 ```
 
+{% aside %}
+{% component-inline-prop propPath=["icon"] %}📢{% /component-inline-prop %}
```

Comment:
This has new syntax now, right? Might need to merge `main` and update this?

Reply:
Ahh thanks for that catch!!